frontend/src/patientApp/timeOfTest/TestResult.test.tsx

```diff
@@ -0,0 +1,100 @@
+import { MockedProvider } from "@apollo/client/testing";
+import { Provider } from "react-redux";
+import configureStore from "redux-mock-store";
+import { Router } from "react-router-dom";
+import { createMemoryHistory } from "history";
+import { render } from "@testing-library/react";
+
+import TestResult from "./TestResult";
+
+const mockStore = configureStore([]);
+
+describe("TestResult", () => {
+  it("should show a positive result", () => {
+    const store = mockStore({
+      plid: "foo",
+      patient: {
+        firstName: "Bob",
+        lastName: "Barker",
+        lastTest: {
+          deviceType: "Testing device",
+          dateTested: "08/27/2020",
+          result: "POSITIVE",
+        },
+      },
+    });
+    const { container, getByText } = render(
+      <Router history={createMemoryHistory()}>
+        <Provider store={store}>
+          <MockedProvider mocks={[]} addTypename={false}>
```

Comment:
Seems like none of the tests in this file should need to be wrapped in a `MockedProvider` since the `TestResults` component doesn't use graphql

Reply:
Aha, good catch! Will remove